package.json

```diff
@@ -161,7 +161,7 @@
     "jest": "jest --notify --watch",
     "e2e-tests": "jest --runInBand --config=jest.config.e2e.js",
     "api-tests": "jest --notify --watch --config=devenv/e2e-api-tests/jest.js",
-    "storybook": "cd packages/grafana-ui && yarn storybook",
+    "storybook": "cd packages/grafana-ui && yarn storybook --ci",
```

Comment:
Any other consequences apart from not opening the browser window?

Reply:
Ops, did not intend to push this. Have not seen any side effect besides not opening a browser window.

Reply:
no worries